go.mod

```diff
@@ -7,7 +7,7 @@ replace (
 	github.com/siderolabs/talos/pkg/machinery => ./pkg/machinery
 
 	// github.com/cilium/ebpf@v0.11.0/internal/unix/types_linux.go:41:36: undefined: linux.BPF_F_KPROBE_MULTI_RETURN
-	golang.org/x/sys => golang.org/x/sys v0.13.0
+	// golang.org/x/sys => golang.org/x/sys v0.13.0
```

Comment:
remove?

Reply:
oh yeah, I found out that it's not needed, but forgot to remove it completely, thanks!